bin/tictactoe

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+puts "Welcome to Tic Tac Toe!"
+Game.start
```

Comment:
Exactly how the bin/tictactoe should look. However, we would prefer to encapsulate CLI logic into a separate class. For example, you could have a file: `lib/tic_tac_toe_cli.rb` with:

```ruby
class TicTacToeCLI
  def start
   # Your game logic
  end
end
```

Separating the execution interface from the implementation of the game allows you to have multiple interfaces - e.g. a web app.